File: NativeAPI/Python/b2_list_unfinished_large_files.py
import requests
import json

api_url = ""  # Provided by b2_authorize_account
account_authorization_token = ""  # Provided by b2_authorize_account
bucket_id = ""  # Provided by b2_list_buckets

response = requests.get(
    f"{api_url}/b2api/v4/b2_list_unfinished_large_files",
    headers={"Authorization": account_authorization_token},
    params={"bucketId": bucket_id}
)
response_data = response.json()
print(json.dumps(response_data, indent=2))

File: NativeAPI/Java/B2ListUnfinishedLargeFiles.java
package com.backblaze.examples;

import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;

public class B2ListUnfinishedLargeFiles {
    public static void main(String[] args) {
        String apiUrl = ""; // Provided by b2_authorize_account
        String accountAuthorizationToken = ""; // Provided by b2_authorize_account
        String bucketId = ""; // Provided by b2_create_bucket or b2_list_buckets

        try (HttpClient client = HttpClient.newHttpClient()) {
            HttpRequest request = HttpRequest.newBuilder()
                    .uri(URI.create(apiUrl + "/b2api/v4/b2_list_unfinished_large_files?bucketId=" + bucketId))
                    .header("Authorization", accountAuthorizationToken)
                    .GET()
                    .build();
            HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
            String jsonResponse = response.body();
            System.out.println(jsonResponse);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
